lib/pure/os.nim

```diff
@@ -2266,17 +2266,21 @@ iterator walkDir*(dir: string; relative = false, checkDir = false):
   ##        / fileA2.txt
   ##
   ## and this code:
-  ##
-  ## .. code-block:: Nim
-  ##     for kind, path in walkDir("dirA"):
-  ##       echo(path)
-  ##
+  runnableExamples("-r:off"):
+    for kind, path in walkDir("dirA"):
+      echo(path)
   ## produce this output (but not necessarily in this order!)::
   ##   dirA/dirB
   ##   dirA/dirC
   ##   dirA/fileA1.txt
   ##   dirA/fileA2.txt
   ##
+  ## It also works in VM and produces similar results like above.
+  runnableExamples("-r:off"):
+    static:
+      for kind, path in walkDir("dirA"):
```

Comment:
but then this actually runs at CT, that's probably not intended here; instead combine with previous example with
```nim
  runnableExamples("-r:off"):
    import std/sequtils
    # note: order is not guaranteed
    # this also works at compile time
    assert toSeq(walkDir("dirA")).join(",") == "dirA/dirB dirA/dirC dirA/dirC dirA/fileA2.txt"
```
which is IMO simpler and no less readable
(we have tests anyways)

Reply:
But `dirA` doesn't exist so no big deal?

Reply:
fixed.